### PR TITLE
Added POST endpoints

### PR DESCRIPTION
**Pull request by LotuxPunk**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "clean": "rm -rf ./node_modules && rm -rf ./packages/*/node_modules",
     "postinstall": "npm run bootstrap",
     "bootstrap": "lerna bootstrap",
-    "initDB": "npm run build && node ./packages/ldes-manager/bin/orm/init.js"
+    "initDB": "npm run build && node ./packages/ldes-manager/bin/orm/init.js",
```
Comment (DemianD) [inline package.json:25]: Maybe it's an idea to use migrations for the database. See https://sequelize.org/master/manual/migrations.html

File: packages/ldes-postgres-connector/template.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+export const template = {
```
Comment (DemianD) [inline packages/ldes-postgres-connector/template.ts]: Maybe you can add a type to this object like so:

```
export const template: IConfigTemplate = {
```

File: packages/ldes-manager/lib/routes/Orchestrator.route.ts [hunk 3]
```diff
@@ -3,12 +3,33 @@ import fp from 'fastify-plugin';
...
+}
+
+const orchestratorBodyJsonSchema = {
+  type: 'object',
+  required: ['name', 'ldes_uri', 'connectorId', 'polling_interval'],
+  properties: {
+    name: { type: 'string' },
+    ldes_uri: { type: 'string' },
+    connectorId: { type: 'number' },
+    polling_interval: { type: 'number' },
```
Comment (DemianD) [inline packages/ldes-manager/lib/routes/Orchestrator.route.ts]: If possible, maybe also use snake_caase for the connectorId. I think I know why it's camel case, but yeah, for and end user one casing type would be easier to work with 😁

File: packages/ldes-postgres-connector/template.ts (hunk 2 above)
Comment (LotuxPunk) [reply packages/ldes-postgres-connector/template.ts]: Done !

File: packages/ldes-manager/lib/routes/Orchestrator.route.ts (hunk 3 above)
Comment (LotuxPunk) [reply packages/ldes-manager/lib/routes/Orchestrator.route.ts]: Done !

File: package.json (hunk 1 above)
Comment (LotuxPunk) [reply package.json:25]: I am not sure on how to do that since I didn't use the `sequelize-cli`

**Pull request merged**